_posts/2019-03-10-moin-ruby-unconf.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Would you be so nice and move this post to a drafts in order to hide it until we have the first real blog post.

Reply:
I changed the title of the MR to make clear this branch should not directly be used. IMHO we should have a real first bog entry with meaningful content and then release the the blog section. A entry in `_drafts` is not a good template